manifests/charts/istio-control/istio-discovery/templates/clusterrole.yaml

```diff
@@ -71,6 +71,7 @@ rules:
     verbs: ["create", "get", "list", "watch", "update"]
 
   # Istiod and bootstrap.
+{{- if or .Values.pilot.env.EXTERNAL_CA (ne .Values.global.pilotCertProvider "istiod") }}
```

Comment:
can we ignore all of "istiod", "custom", "none"?

Reply:
Done